archive/m/moonscript/README.md

```diff
@@ -0,0 +1,20 @@
+# Sample Programs in MOONSCRIPT
```

Comment:
I'm not 100% sure, but I think the official spelling is "MoonScript". Could you clarify this? imo it's better to consistently use the official capitalization

Reply:
yeah thats correct!